packages/dev/core/src/scene.ts

```diff
@@ -256,6 +268,34 @@ export class Scene extends AbstractScene implements IAnimatable, IClipPlanesHold
         return this._imageProcessingConfiguration;
     }
 
+    private _performancePriority = ScenePerformancePriority.BackwardCompatible;
+    /**
+     * Gets or sets a value indicating how to treat performance relatively to ease of use and backward compatibility
+     */
+    public get performancePriority() {
+        return this._performancePriority;
+    }
+
+    public set performancePriority(value) {
+        if (value === this._performancePriority) {
+            return;
+        }
+
+        this._performancePriority = value;
+
+        switch (value) {
+            case ScenePerformancePriority.Intermediate:
+                this.skipPointerMovePicking = true;
+                this.autoClear = false;
+                break;
+            case ScenePerformancePriority.Aggressive:
+                this.autoClear = false;
+                this.skipPointerMovePicking = true;
+                this.skipFrustumClipping = true;
+                break;
+        }
+    }
+

```

Comment:
Maybe rewrite it like this to avoid duplicated code:
```typescript
switch (value) {
    case ScenePerformancePriority.Aggressive:
        this.skipFrustumClipping = true;
        // note: intentional fall-through
    case ScenePerformancePriority.Intermediate:
        this.skipPointerMovePicking = true;
        this.autoClear = false;
        break;
}
```

Reply:
good one